lib/types.ts

```diff
@@ -32,3 +32,8 @@ export interface ResultMessage {
    isError: boolean;
    message: string;
 }
+
+export interface FileObject {
```

Comment:
may have missed it but did this get used anywhere

Reply:
Yep to declare the attachments type